test/common/ops/releaseBoth.js

```diff
@@ -78,6 +78,24 @@ describe('shared.ops.releaseBoth', () => {
     expect(user.items.currentMount).to.be.empty;
   });
 
+  it('leaves quest pets and mounts equipped', () => {
+    let questAnimal = 'Gryphon-Base';
+    user.items.currentMount = questAnimal;
+    user.items.currentPet = questAnimal;
+    user.items.pets[questAnimal] = 5;
+    user.items.mounts[questAnimal] = true;
+    releaseBoth(user);
+
+    expect(user.items.currentMount).to.equal(questAnimal);
+    expect(user.items.currentPet).to.equal(questAnimal);
+  });
+
+  it('removes currentMount', () => {
```

Comment:
Probably should specify that this condition is met when the mount is a drop mount. You could also include assertions about currentPet in this test.